Trilinos quicktest fail when compiled against Trilinos 17.*

As evidenced here: https://cdash.dealii.org/tests/1044871
```
/srv/temp/buildtest-imWXlU3G/dealii/tests/quick_tests/step-trilinos.cc:157:21: error: no member named 'SolverCG' in namespace 'dealii::TrilinosWrappers'
  157 |   TrilinosWrappers::SolverCG                cg_solver(solver_control);
      |                     ^~~~~~~~
/srv/temp/buildtest-imWXlU3G/dealii/tests/quick_tests/step-trilinos.cc:160:3: error: use of undeclared identifier 'cg_solver'
  160 |   cg_solver.solve(A, x, b, preconditioner);
      |   ^~~~~~~~~
2 errors generated.
```

--- a/tests/quick_tests/step-trilinos.cc
+++ b/tests/quick_tests/step-trilinos.cc
@@ -32,8 +32,8 @@
 
 #include <deal.II/lac/dynamic_sparsity_pattern.h>
 #include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/solver_cg.h>
 #include <deal.II/lac/trilinos_precondition.h>
-#include <deal.II/lac/trilinos_solver.h>
 #include <deal.II/lac/trilinos_sparse_matrix.h>
 #include <deal.II/lac/trilinos_vector.h>
 
@@ -154,7 +154,7 @@ void
 LaplaceProblem::solve()
 {
   SolverControl                             solver_control(1e03, 1e-03);
-  TrilinosWrappers::SolverCG                cg_solver(solver_control);
+  SolverCG<TrilinosWrappers::MPI::Vector>   cg_solver(solver_control);
   TrilinosWrappers::PreconditionBlockJacobi preconditioner;
   preconditioner.initialize(A);
   cg_solver.solve(A, x, b, preconditioner);

